code/modules/projectiles/guns/hookshot.dm

```diff
@@ -506,3 +506,140 @@
 	icon_state = "vampkiller_true"
 	hooktype = /obj/item/projectile/hookshot/whip/vampkiller/true
 
+//Windup-Boxes/////////////////////////////////////////////////////
+
+/obj/item/weapon/gun/hookshot/whip/bootbox
+	name = "boot-in-a-box"
+	icon = 'icons/obj/wind_up.dmi'
+	icon_state = "bootbox-0"
+	item_state = "bootbox"
+	desc = "A box with a spring-loaded boot inside. There is a crank attached to wind it up."
+	fire_action = "activate"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/guns_experimental.dmi', "right_hand" = 'icons/mob/in-hand/right/guns_experimental.dmi')
+	clumsy_check = 0 //Just makes sense
+	force = 5
+	maxlength = 4 //whips are weird and this ends up being 2
+	hooktype = /obj/item/projectile/hookshot/whip/bootbox
+	var/windUp = 0 //amount of times cranked
+	var/maxWindUp = 16 //threshold for overwind
+	var/overWind = 0 //warning/delay system
+	var/state = 0 //Icon changes with each crank, stolen from the crank cell charger
+	var/springForce = 0 //makes the big kicks
+
+/obj/item/weapon/gun/hookshot/whip/bootbox/update_icon()
+	icon_state = "bootbox[state ? "-1" : "-0"]"
+
+/obj/item/weapon/gun/hookshot/whip/bootbox/attack_self(mob/user)
+	state = !state
+	update_icon()
+	if(prob(windUp*springForce)) //prob 0 before they start forcing it, perfectly safe
+		explosion(src.loc,-1,0,1)
+		qdel(src)
+		return
+	if(windUp >= maxWindUp && overWind <3) //give them a chance to stop winding in the safety zone. Also largely to delay on-the-run use.
+		user.delayNextAttack(20)
+		playsound(src,'sound/weapons/smash.ogg', 25, 1)
+		overWind++
+		to_chat(user, "<span class='notice'>The crank will barely move.</span>")
+		return
+	if(overWind >=3) //we are no longer in the safety zone. You get one free one before boom risk
```

Comment:
because of how you've written this, if they have overWind = 2, it will first pass the first if and increase to 3, then trigger the second if

what I would do is remove overwind < 3 from the first if and instead nest the overwind into the other if

```
if(windUp >= maxWindUp) //give them a chance to stop winding in the safety zone. Also largely to delay on-the-run use.
		if(overWind<3)
				user.delayNextAttack(20)
				playsound(src,'sound/weapons/smash.ogg', 25, 1)
				overWind++
				to_chat(user, "<span class='notice'>The crank will barely move.</span>")
				return
		else
				to_chat(user, "<span class='notice'>With great difficulty you get the crank moving.</span>")
				etc
```